cogsmisc/customization.py

```diff
@@ -369,6 +369,24 @@ def can_edit_servaliases(ctx):
         parsed = await char.parse_cvars(teststr, ctx)
         parsed = clean_content(parsed, ctx)
         await ctx.send(f"{ctx.author.display_name}: {parsed}")
+                       
+    @commands.command()
+    async def tembed(self, ctx, *, teststr):
+	    """Parses `str` as if it were in an alias, for testing, then creates and prints an Embed.
+	    Arguments: -title [title]
+	    -desc [description text]
+	    -thumb [image url]
+	    -image [image url]
+	    -footer [footer text]
+	    -f ["Field Title|Field Text"]
+	    -color [hex color]
+	    -t [timeout (0..600)]
+	    """
+
+        char = await Character.from_ctx(ctx)
+        parsed = await char.parse_cvars(teststr, ctx)
+
+        return await ctx.invoke(self.bot.get_command('embed'), args=parsed)
```

Comment:
`self.bot.get_command("embed")` should always work, but it's probably a good idea to have a handler for if it returns `None`